aws-auth-cognito/src/test/java/com/amplifyframework/auth/cognito/featuretest/generators/JsonGenerator.kt

```diff
@@ -61,6 +61,6 @@ object JsonGenerator {
 }
 
 fun main() {
-    cleanDirectory()
+    // cleanDirectory()
```

Comment:
Do we not need to clean outdated json test files?

Reply:
This step is manually run anyway so does not matter. This function also clears out the configuration files which are not regenerated so this has to be tweaked.